code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+		icon_state = "neck"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	neckthick
+		name = "Neck Cover (Thick)"
+		icon_state = "neckthick"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	nosestripe
+		name = "Nose Stripe"
+		icon_state = "nosestripe"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	nosetape
+		name = "Nose Tape"
+		icon_state = "nosetape"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Vaurca")
+
+	scratch_abdomen_left
+		name = "Scratch, Abdomen Left"
+		icon_state = "scratch_abdomen_l"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_abdomen_right
+		name = "Scratch, Abdomen Right"
+		icon_state = "scratch_abdomen_r"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_abdomen_small_left
+		name = "Scratch, Abdomen Small Left"
+		icon_state = "scratch_abdomensmall_l"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_abdomen_small_right
+		name = "Scratch, Abdomen Small Right"
+		icon_state = "scratch_abdomensmall_r"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_back
+		name = "Scratch, Back"
+		icon_state = "scratch_back"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_chest_left
+		name = "Scratch, Chest (Left)"
+		icon_state = "scratch_chest_l"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	scratch_chest_right
+		name = "Scratch, Chest (Right)"
+		icon_state = "scratch_chest_r"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	skull_paint
+		name = "Skull Paint"
+		icon_state = "skull"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_belly
+		name = "Tattoo (Belly)"
+		icon_state = "tat_belly"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_campbell_leftarm
+		name = "Tattoo (Campbell, Left Arm)"
+		icon_state = "tat_campbell"
+		body_parts = list("l_arm")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_campbell_rightarm
```

Comment:
Metal Gear Solid Reference

Reply:
> This is one of those things where it was actually named such as-is and can even be found named so in quite a few other servers that have them.
I wish to understand what's the risk/consequence/problem in having a tattoo be named, on the menu, POSSIBLY after a character, if you would care to grant me such a thing?

> Regardless, I can try to rename it something else.

Reply:
Its moreso still visually immersion breaking for people, however renaming it should suffice since making it out is hard